Utility/GroupSorting.FCMacro

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# (c) 18Turbo, 2022
+
+__Name__ = 'GroupSorting (Ordenación)'
+__Comment__ = 'The macro sorts the groups and pieces of a piece.\nSelect groups with <CTRL> key, and execute the macro.'
+__Author__ = '18Turbo'
+__Version__ = '0.1.0'
+__Date__ = '2022-10-04'
+__License__ = 'LGPL-2.0-or-later'
+__Web__ = 'https://github.com/18turbo/OrdenacionRapidaFreeCAD'
+__Wiki__ = 'https://github.com/18turbo/OrdenacionRapidaFreeCAD'
+__Icon__ = 'GroupSorting.svg'
+__Help__ = ''
+__Status__ = 'beta'
+__Requires__ = 'FreeCAD >= v0.20'
+__Contact__ = 'https://github.com/18turbo'
+__Communication__ = ''
+__Files__ = 'GroupSorting.svg'
+
+
+# (Spanish:)
+#
+# Autor:            18Turbo
+# Fecha Incial:     22-07-2022
+# Versión:          0.1.0 BETA
+#   
+# Funcionalidad:   Ordena grupos según sean seleccionados.
+#
+# Notas Importantes:     hay que saber que los grupos sin padre tienen un orden específico en FreeCAD y no se pueden ordenar con este script
+#                   porque aunque se hace el proceso correcto, FreeCAD no permite la ordenación de estos de una forma simple y escapa
+#                   a los propósitos de este script.
+#
+#
+# Notas de versiones:
+# v.0.1.0:        Adaptación al Addon Manager
+# v.0.0.5:      Depurado para que funcione con grupos sin padre, es decir, con el documento como padre, aunque FreeCAD no lo permita. Así,
+#               si en una versión superior se permite, el script no hay que modificarlo.
+# v.0.0.4:      Versión Alfa funcional.
+#
+#
+#
+
+import FreeCAD as app
+import FreeCADGui as gui
+
+cIntro = '\n'
```

Comment:
Please give a more explicit name. Here, we don't know the introduction of what and where the variable could be used.